src/lambert-coordinaat.js

```diff
@@ -0,0 +1,61 @@
+class LambertCoordinaat {
+  
+  /**
+   * Lamber Coördinaat regular expression. Allows:
+   * - unlimited spaces before the start of the coordinate
+   * - optional left parenthesis '('
+   * - mandatory x-coordinate, which can be a decimal (with a precision of 2) or integer number of 1-6 digits
```

Comment:
Waarom de beperking in aantal decimalen? Gaat het bvb niet werken als er mensen "preciezere" coordinaten dan van ergens copy-pasten?

Reply:
Moet uw regex niet eerder zeggen dat het stuk "na de komma" optioneel is, ipv daar nu met die "of" te werken?

Reply:
Had me gebaseerd op het voorbeeld uit de Topomap van het Nationaal Geografisch Instituut.
Daar staan ze volgende conventie toe:

<img width="1434" alt="Schermafbeelding 2021-01-26 om 17 42 40" src="https://user-images.githubusercontent.com/1375279/106315094-0764c780-626b-11eb-8077-e684faf45ab3.png">

Maar nu kan je dus perfect een decimaal of geheel getal invullen. Misschien moet ik nog rekening houden met coördinaten die meer cijfers na de komma kunnen bevatten en dan enkel tot op 2 de waarde nemen. Heb eigenlijk geen weet of een grotere precisie kan voorkomen. Ging ervan uit dat wat ze hierboven opgaven de norm was.

Reply:
Ja, afronden lijkt me idd geen probleem, maar ingave met meer decimalen zou ik idd toelaten.